src/org/usfirst/frc/team467/robot/RobotMap.java

```diff
@@ -66,6 +66,8 @@ public static void init(RobotID id) {
 			HAS_ELEVATOR = false;
 			HAS_GRABBER = false;
 			HAS_RAMPS = false;
+			HAS_LEFT_RAMP = false;
+			HAS_RIGHT_RAMP = false;
```

Comment:
Do we still need HAS_RAMPS? Or if we do it could be defined as:
`HAS_RAMPS = HAS_LEFT_RAMP || HAS_RIGHT_RAMP;`

I think you don't need it in RobotMap anymore, and the users of this data can figure it out.